extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -107,8 +109,7 @@ public YoutubeStreamExtractor(StreamingService service, LinkHandler linkHandler)
     public String getName() throws ParsingException {
         assertPageFetched();
         try {
-            return playerResponse.getObject("videoDetails").getString("title");
-
+            return getString(playerResponse.getObject("microformat").getObject("playerMicroformatRenderer").getObject("title"), "simpleText");
```

Comment:
what is the reason for changing the order?

Reply:
> > The only related problem is channel subscription count, so I fixed it this way: if the content language is not English and if the sub count is shortened, it makes a new request in English and get channel sub count.
> 
> What is the exact problem? Do you get the wrong format or no result?

The problem is: Months ago, YouTube shortened sub count for channels, leading to no exact number. As it's shortened, it gives 250k, 1M… but in other languages, it could be 250 k (french), 250 somewhat (with space), and then we only gather the number, leading to https://github.com/TeamNewPipe/NewPipe/issues/2632
Enforcing english fixed this. But then broke title & description in the wrong language. Making a new request in English if the number is possibly wrong is the solution I thought

Reply:
The reason for changing from videoDetails to microformat in JSON:
videoDetails gives the description in the original language.

The reason for changing to JSON first:
Bullet list are shown wrong with HTML.

See https://www.youtube.com/watch?v=nQTvw4_pzGk
Screenshot with official build:
<img src="https://i.ibb.co/6W7rmZY/Screenshot-20200215-132344-New-Pipe-PR-2309.png" width="200">
and this screenshot (made with the test apk & this PR)
<img src="https://i.ibb.co/7Wvw8s7/Screenshot-20200215-132313-New-Pipe-Debug.png" width="200">

Reply:
ok. thanks. Can you add this info as a comment?

Reply:
yeah